### PR TITLE
Part1  doc improvement

### PR DESCRIPTION
**Pull request by Hard-Coder05**
Fixes: Issue #648 (https://github.com/appsmithorg/appsmith/issues/648) 
All the changes mentioned are completed!

### CODE REVIEW

**Comment by Hard-Coder05** [comment]
@Nikhil-Nandagopal @mohanarpit I have raise th PR please review it once!

File: faq.md [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 Appsmith isn't built for complicated UI interactions that are common to e-commerce or other consumer-facing websites. It works great for tools internal to your team or organization but you wouldn't get very far trying to build an attractive consumer website. For those use cases, we'd recommend you try [Wix](www.wix.com), [Squarespace](www.squarespace.com), or [Webflow](www.webflow.com). You can also perform some simple data visualization & charting on Appsmith but for more in-depth business intelligence & analytics, we'd recommend [Tableau](https://www.tableau.com/), [Looker](https://looker.com/), or [Redash](https://redash.io/).
```
Comment (Nikhil-Nandagopal) [inline faq.md:31]: This seems incorrect

File: faq.md [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
+Appsmith safely encrypts all your database credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Appsmith is an open source framework and can be fully audited and deployed on-premise to ensure none of your data leaves your VPC.
 
-## What if I get stuck building an app on Appsmith?
+## What if I get stuck while building an app on Appsmith?
 
 We sincerely hope this never happens to anyone but we acknowledge that there can be road blocks. At this juncture, we're working on a way for you to be able to write custom react components and plugins to unblock you from any platform limitations. In the meantime, if the absence of any feature is limiting you, you can write to [support@appsmith.com](mailto:%20support@appsmith.com) or reach out to us on our [discord server](https://discord.com/invite/rBTTVJp) 
 
-## What can't I do on Appsmith?
+## Like with every GOD particle, Appsmith also has its own limitations?
```
Comment (Nikhil-Nandagopal) [inline faq.md:28]: Let's not make this change

File: quick-start.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Appsmith stands for speed and getting started with Appsmith is just as fast.
+description: Appsmith stands for speed and getting started with Appsmith is just as fast as Usain Bolt.
```
Comment (Nikhil-Nandagopal) [inline quick-start.md:2]: Please skip this change

File: faq.md [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
 Appsmith is the first open-source visual framework for building internal tools. We make building internal tool building easy by giving you building blocks \(widgets, APIs, Queries\) which you can configure and connect together while the framework does most of the heavy lifting
 
 ## Can I write code in Appsmith?
 
-Yes, absolutely. You can write JS code anywhere inside the product within `{{ }}`. We know how complex and intricate business logic can be and trying to build it without code can get excruciating. Appsmith bridges mundane API and UI configurations with powerful code expressions to let you build apps the way you see fit.
+Yes, absolutely. You can write JS code anywhere inside the product within `{{ }}`. We know how complex and intricate business logic can be and trying to build it without code can get excruciating.  Appsmith bridges mundane API and UI configurations with powerful code expressions to let you build the apps the way you see them fit.
```
Comment (Nikhil-Nandagopal) [inline faq.md:18]: This change is incorrect

File: faq.md [hunk 5]
```diff
@@ -1,31 +1,33 @@
 ---
-description: You may have these questions and we want to answer.
+description: You may have these questions and we would like to answer them.
 ---
 
 # FAQs
 
-## What type of apps are built on Appsmith?
+## What kind of apps can be built on Appssmith?
```
Comment (Nikhil-Nandagopal) [inline faq.md:7]: Appsmith is misspelled

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
@Hard-Coder05 i've requested a few changes. Also please run this through grammarly

**Comment by Hard-Coder05** [comment]
@Nikhil-Nandagopal I have made the required changes.

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Comment by Hard-Coder05** [comment]
@Nikhil-Nandagopal what about the hacktoberfest swags?...

**Comment by areyabhishek** [comment]
@Hard-Coder05 Thanks for contributing! Could you fill out your information in this form? https://airtable.com/shrZNbhUJyoVkYVws We'll be shipping swag in a couple of weeks.

**Pull request merged**